lib/perl/Genome/Model/Build.pm

```diff
@@ -1309,12 +1309,18 @@ sub _kill_job {
     return 1;
 }
 
+sub process {
```

Comment:
It might be nice for this to be a property of builds, like:

```perl
process => {
  is => 'Genome::Model::Build::Process',
  reverse_as => 'build',
}
```

Then we could let `genome model build view` work given a process ID or something similar.